microsoft/PowerToys
# [PowerRename] Sort files before adding them to list view

## Summary of the Pull Request

**What is this about:**
Win API enumerate files in round robin fashion, starting with file which was right click-ed. This results in bad user experience in PowerRename where files appear non-sorted (in different order than shown in Explorer). This is fixed by "hacky" sorting files before adding them to list view.

**What is include in the PR:** 

**How does someone test / validate:** 
 - Install PT 0.43
 - Select multiple files and open PowerRename
 - Observe that files in listview start with file which received right click
 - Install PT using this branch
 - Repeat the steps and observe that files are sorted in list view regardless where right click was pressed

## Quality Checklist

- [x] **Linked issue:** #11640 

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
